valhalla/mjolnir/osmnode.h

```diff
@@ -94,7 +94,8 @@ struct OSMNode {
    */
   midgard::PointLL latlng() const {
     // if either coord is borked we return invalid pointll
-    if (lng7_ == -1 || lat7_ == -1)
+    if (lng7_ == std::numeric_limits<uint32_t>::max() ||
+        lat7_ == std::numeric_limits<uint32_t>::max())
```

Comment:
this is not right! look at the method above, it explicitly sets them to `-1` and if they were out of range they stay `-1`

Reply:
its odd that assignment is fine but comparison is not

Reply:
Odd indeed. Updated the assignment as well.